Cubical/Foundations/Groupoid.agda

```diff
@@ -0,0 +1,100 @@
+{-
+
+Proof of some groupoid laws for equality
+
+-}
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Foundations.Groupoid where
+
+open import Cubical.Core.Primitives
+open import Cubical.Core.Prelude
+
+-- shortcut for the composition
+
+_⋆_ : {ℓ : Level} → {A : Set ℓ} → {x y z : A} → (x ≡ y) → (y ≡ z) → (x ≡ z)
```

Comment:
This should be upstreamed, maybe this should even be the name of `compPath`

Reply:
The HoTT book uses a small black square, maybe it would be better to pick the same notation. However, I am not sure there is an easily accessible small black square in emacs' TeX input mode (■  is a bit large ?).

Reply:
I think "\." is quite nice (I think of it as multiplication in the infty groupoid)

Reply:
I changed it to "⋅" (\cdot) in the files involved in this PR, I guess upstreaming it should be the job of a future PR ?

Reply:
That sounds like a good idea